tools/rostest/test/hztest.test

```diff
@@ -6,19 +6,6 @@
   <param name="hztest1/hzerror" value="0.5" />
   <param name="hztest1/test_duration" value="5.0" />    
   <param name="hztest1/wait_time" value="21.0" />    
-  <test test-name="hztest_test" pkg="rostest" type="hztest" name="hztest1" />
+  <test test-name="hztest_test" pkg="rostest" type="hztest" name="hztest1" retry="3" />
 
-  <!-- Below also works:
```

Comment:
pointless comment, yes this looks like it also works, but why should this be in here?